src/components/visualizations/chart/chartOptions/test/comboChartOptions.spec.ts

```diff
@@ -36,3 +37,50 @@ describe("getComboChartSeries", () => {
         },
     );
 });
+
+describe("getComboChartStackingConfig", () => {
+    it("should return null when there is no stacking config", () => {
+        expect(getComboChartStackingConfig({}, [], null)).toBe(null);
+    });
+
+    it("should return default 'percent' stack value", () => {
+        expect(
+            getComboChartStackingConfig(
+                { stackMeasures: true },
+                [
+                    {
+                        yAxis: 0,
+                        type: "column",
+                    },
+                    {
+                        yAxis: 1,
+                        type: "line",
+                    },
+                ],
+                PERCENT_STACK,
+            ),
+        ).toBe(PERCENT_STACK);
+    });
+
+    it.each([[NORMAL_STACK, true], [null, false]])(
+        "should return %s stack value when 'Stack Measures' config is %s",
+        (stackValue: string, stackMeasures: boolean) => {
```

Comment:
`stackValue` can be `null`, using `stackValue: string` here is not correct

Reply:
done